### DISCUSSION TITLE
DeepSleepTime wakes up but does not repeat the cycle...

### DISCUSSION THREAD

**Discussion by saintofinternet** [arendst/Tasmota#12439] [Q&A] [answered]
I have followed all the instructions as per Tasmota page (https://tasmota.github.io/docs/DeepSleep/) and wired up RST and GPIO16 of my WEMOS D1 Mini.

When I give the command in console for DeepSleepTime 120 and Telemetery is 10 the device goes to sleep for 120 seconds... wakes up and performs the desired actions but then does not go in DeepSleep again... 

It keeps sending telemetry data every 10 seconds but does not go into DeepSleep again.

Am I going wrong anywhere?

**Comment by barbudor** [comment, marked as answer]
We can't guess anything without logs
Please provide logs from the wake-up for up to a few telemetry messages
Please provide also the output of commands 'module', 'template' and 'gpio all'

**Reply by barbudor** [reply]
As the device properly goes to sleep the 1st time, wakes up but doesn't go to sleep again, the most likely cause are:
- not getting proper time from NTP after wake up => fix wifi or NTP issues
- you have defined a pin as `DeepSleep (4096)`, this pin is tied down and prevent going back to DeepSleep => remove this pin for now
